website/content/en/docs/building-operators/golang/operator-scope.md

```diff
@@ -89,42 +97,53 @@ in `role_binding.yaml` are used to grant the operator permissions to access and
 For the purposes of this doc, the other RBAC manifests `<kind>_editor_role.yaml`, `<kind>_viewer_role.yaml`, 
 and `auth_proxy_*.yaml` are not relevant to changing the operator's resource permissions.
 
-### Changing the permissions 
+### Changing the permissions to Namespaced
+
+- Inform the Namespace to the [Manager][ctrl-manager] 
+
+By default, the [Manager][ctrl-manager] in the `main.go` is initialized with no Namespace option specified which mean that will watch all Namespaces. Update the `NewManager` to inform the Namespace, in our `Memcahced` example it would like: 
```

Comment:
(Clarifying it with KB book) Nit: 
By default, the [Manager][ctrl-manager] does not have any namespace specified in `main.go`, and hence it will watch all the namespaces. In order to restrict the controllers to watch a specific namespace, specify it while creating the `manager`.